From 7cfca46c302776a5d666e1f3e05b2b288af9fef2 Mon Sep 17 00:00:00 2001
From: suqing guo <suqing.guo@unisoc.com>
Date: Tue, 18 Oct 2022 15:37:49 +0800
Subject: [PATCH] Bug #2052013 add BT selinux for getopt

[root cause  ]: add BT selinux for getopt
[changes     ]: add BT selinux for getopt
[side effects]: no
[self test   ]: test
[download normally]: yes
[power on/off normally]: yes
[do common repository/branch inspection]: yes
[is there dependence]: no
[confirm dependent commit]: no
[board]: 9863
[test case]: test
[reviewers]: chao.meng

commit_template_version:v1
---
 policy/modules/roles/sysadm.if | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/roles/sysadm.if b/policy/modules/roles/sysadm.if
index 9ba0c24d4..c6ad1017a 100755
--- a/policy/modules/roles/sysadm.if
+++ b/policy/modules/roles/sysadm.if
@@ -499,7 +499,7 @@ interface(`sysadm_sysadm_t_bluetooth_socket',`
          type sysadm_t;
     ')
 
-    allow $1 sysadm_t:bluetooth_socket {write listen read create bind ioctl setopt};
+    allow $1 sysadm_t:bluetooth_socket {write listen read create bind ioctl setopt getopt connect accept getattr};
 ')
 
 interface(`sysadm_dri_device_t_chr_file',`
-- 
2.17.1

